test/ValidationModule/CodeListInternal.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity ^0.8.17;
 
-abstract contract CodeList {
+abstract contract CodeListInternal {
```

Comment:
Should we keep the name `CodeList` and move it into its own director `ValidationModule/Internal`?